.ci/openshift-ci/Dockerfile

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+# Dockerfile to bootstrap build and test in openshift-ci
+FROM registry.ci.openshift.org/openshift/release:golang-1.15
+SHELL ["/bin/bash", "-c"]
+# Install yq, kubectl, chectl cli used by olm/olm.sh script.
```

Comment:
```suggestion
# Install yq, kubectl, chectl cli used by olm/olm.sh script.
```
what is olm.sh? Is it a way we're going to install Che for Dashboard on OpenShift CI?

Reply:
I am thinking.... So , actually it  was grabbed from other project. And I do not know which approach will be more preferable for us for CHE instalation. Actually, I am going to disscuss it with you. It just like init commit for laucnching and debugging the job. It will be changed according to our needs